### PR TITLE
[bitnami/minio] Be able to define custom secretKeys instead of root-user and root-password

### PR DESCRIPTION
**Pull request by alexhfmnn**
### Description of the change

Currently, when using `.auth.existingSecret`, you are forced to use `root-user` and `root-password` as keys for the Minio root user's username and password within the defined secret. This PR adds the possibility - analogous to the chart of bitnami/postgresql - to define own SecretKeys within the values.

### Benefits

Adds adds more flexibility to the Chart

### Applicable issues

- fixes #

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/minio/templates/_helpers.tpl [hunk 1]
```diff
@@ -85,6 +85,36 @@ Get the credentials secret.
...
+{{- if .Values.auth.existingSecret -}}
+    {{- if .Values.auth.rootUserSecretKey -}}
+        {{- printf "%s" (tpl .Values.auth.rootUserSecretKey $) -}}
+    {{- else -}}
+        {{- "root-user" -}}
+    {{- end -}}
+{{- else -}}
+    {{- "root-user" -}}
+{{- end -}}
+{{- end -}}
```
Comment (jotamartos) [inline bitnami/minio/templates/_helpers.tpl:101]: We can simplify this method, what do you think?

```suggestion
{{- define "minio.rootUserKey" -}}
{{- if and .Values.auth.existingSecret .Values.auth.rootUserSecretKey -}}
    {{- printf "%s" (tpl .Values.auth.rootUserSecretKey $) -}}
{{- else -}}
    {{- "root-user" -}}
{{- end -}}
```

File: bitnami/minio/templates/_helpers.tpl [hunk 2]
```diff
@@ -85,6 +85,36 @@ Get the credentials secret.
...
+{{- if .Values.auth.existingSecret -}}
+    {{- if .Values.auth.rootPasswordSecretKey -}}
+        {{- printf "%s" (tpl .Values.auth.rootPasswordSecretKey $) -}}
+    {{- else -}}
+        {{- "root-password" -}}
+    {{- end -}}
+{{- else -}}
+    {{- "root-password" -}}
+{{- end -}}
+{{- end -}}
```
Comment (jotamartos) [inline bitnami/minio/templates/_helpers.tpl:116]: Same as above

**Review by jotamartos — CHANGES_REQUESTED** [review]
Thanks for your contribution. Can you take a look at the changes I proposed? Thanks

File: bitnami/minio/templates/_helpers.tpl (hunk 1 above)
Comment (alexhfmnn) [reply bitnami/minio/templates/_helpers.tpl:101]: Thanks, you're completely right

File: bitnami/minio/templates/_helpers.tpl (hunk 2 above)
Comment (alexhfmnn) [reply bitnami/minio/templates/_helpers.tpl]: Resolved both

**Review by jotamartos — APPROVED** [review]
Thanks for your contribution

File: bitnami/minio/templates/_helpers.tpl [hunk 3]
```diff
@@ -85,6 +85,28 @@ Get the credentials secret.
...
+{{- else -}}
+    {{- "root-user" -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Get the root password key.
+*/}}
+{{- define "minio.rootPasswordKey" -}}
+{{- if (.Values.auth.existingSecret) (.Values.auth.rootPasswordSecretKey) -}}
```
Comment (jotamartos) [inline bitnami/minio/templates/_helpers.tpl:103]: Typo here 

```suggestion
{{- if and (.Values.auth.existingSecret) (.Values.auth.rootPasswordSecretKey) -}}
```

**Review by jotamartos — APPROVED** [review]

**Pull request merged**